src/schema/submission.yaml

```diff
@@ -1048,14 +1052,15 @@ slots:
   ancestry_terms:
     description: >-
       A person's descent or lineage from a population.
+      The Ancestry is captured using a concept from the Human Ancestry Ontology "ancestry category" (HANCESTRO:0004) branch (e.g., African, European, Oceanian).
```

Comment:
Can we shorten the text a little bit? It would be nice to have less text on the excel sheets. Maybe smt like this "A person's lineage, defined by the Human Ancestry Ontology, HANCESTRO:0004 (e.g., African, European, or Oceanian)."